src/hint_processor/uint_utils.zig

```diff
@@ -0,0 +1,37 @@
+const std = @import("std");
+const Felt252 = @import("../math/fields/starknet.zig").Felt252;
+const Int = std.math.big.int.Managed;
+pub fn split(num: Int, comptime N: usize, num_bits_shift: usize) ![N]Felt252 {
+    const allocator = std.heap.page_allocator;
```

Comment:
Here can you pass this allocator as an argument of the function so that the user is free to choose the allocator of his choice. For unit tests, feel free to use `std.testing.allocator`.